eradiate/scenes/measure/_distantsensorarray.py

```diff
@@ -0,0 +1,218 @@
+from __future__ import annotations
+
+from typing import Dict, List, Tuple
+
+import attr
+import numpy as np
+import xarray as xr
+
+from ._core import SensorInfo, measure_factory
+from ._distant import DistantMeasure
+from ..illumination import DirectionalIllumination
+from ...attrs import documented, parse_docs
+from ...units import symbol
+from ...units import unit_context_kernel as uck
+
+
+@measure_factory.register(type_id="distantarray")
```

Comment:
Consistency: I'd rather have this called `distant_array` (better correspondence between camel and snake case).